File: backend/src/main/java/com/team05/petmeeting/domain/adoption/dto/AdoptionReviewReq.java
package com.team05.petmeeting.domain.adoption.dto;

import com.team05.petmeeting.domain.adoption.entity.AdoptionStatus;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class AdoptionReviewReq {
    private AdoptionStatus status;
    private String rejectionReason;
}

File: backend/src/main/java/com/team05/petmeeting/domain/adoption/dto/AdoptionReviewReq.kt
package com.team05.petmeeting.domain.adoption.dto

import com.team05.petmeeting.domain.adoption.entity.AdoptionStatus

data class AdoptionReviewReq @JvmOverloads constructor(
    val status: AdoptionStatus? = null,
    val rejectionReason: String? = null,
)
